app_dart/lib/src/service/config.dart

```diff
@@ -86,8 +86,7 @@ class Config {
   // GoB retry options
   static const RetryOptions gobRetryOptions = RetryOptions(
     maxAttempts: 4,
-    maxDelay: Duration(seconds: 1),
-    delayFactor: Duration(milliseconds: 300),
+    delayFactor: Duration(milliseconds: 500),
```

Comment:
what is your expected total wait for this?

Reply:
The wait is 500, 1000, 1500 milliseconds. Based on the info you gave for 1 minute this should be on either side of that. The maxDelay doesn't make sense with this as we would just be limiting the delay factor

Reply:
You specified milliseconds not minutes?

Reply:
Ah sorry. I misread what you said about the wait time.